Clinical-Genomics/cg
# show which samples are external

This PR shows which samples are external in the clinical-api

Contact: @emiliaol 

How to test:
0. install clinical-api on clinical-db stage 
1. login to clinical-api-stage
1. got to the sample view

Expected outcome:
There should be an column for is_external with some samples with a tick and some with a minus symbol

- [x] Code reviewed by @b4ckm4n 
- [x] Tested by @b4ckm4n 
- [x] Deploy OK:d by @patrikgrenfeldt 

This is a minor **version bump** because it expands the interface in a backwards compatible manner
